kotlin-analysis-core/src/main/kotlin/org/jetbrains/research/ml/kotlinAnalysis/Recourse.kt

```diff
@@ -0,0 +1,26 @@
+package org.jetbrains.research.ml.kotlinAnalysis
+
+import org.jetbrains.research.ml.kotlinAnalysis.util.getPrintWriter
+import java.io.PrintWriter
+import java.nio.file.Path
+
+/** Interface for any recourse, which must been initialized before and closed after use. **/
+interface Resource : AutoCloseable {
```

Comment:
I am not sure about this name...it is `Worker` or something like this. Not? @annav1asova what do you think?

Reply:
After discussion we came to `ResourceManager`